canonical-web-and-design/ubuntu.com
# Account users/Add labels to mobile table view

## Done

- add labels to mobile table view
  - create `UsersTableHeader` and `UserCell` with typed `label` prop to make sure they use the same labels

## QA

- go to https://ubuntu-com-10445.demos.haus/advantage/users?test_backend=true and login
- make sure that labels are visible on mobile breakpoint


## Issue / Card

https://github.com/canonical-web-and-design/commercial-squad/issues/282

## Screenshots

### Before
![image](https://user-images.githubusercontent.com/7452681/134333915-899ed53b-8acd-4d62-a7b9-b5b0a23f6a2f.png)

### After
![image](https://user-images.githubusercontent.com/7452681/134333928-0b902c0c-d711-4cdf-b6e9-c648e6ce7909.png)
